openedx/features/custom_fields/multiselect_with_other/helpers.py

```diff
@@ -1,7 +1,15 @@
+""" This module contains helper functions for multi_select_With_other"""
+
 from openedx.features.custom_fields.multiselect_with_other.constants import OTHER_FIELD_CHECKBOX_VALUE
 
 
 def add_other_field_in_choices(choices):
+    """ This function to separate other's value from list of choices
+    :param choices: list of choices
+    :type choices: list
+    :return: list with other field.
+    :rtype: list
```

Comment:
Can choices be a `dict` or `list` or `tuple` instead of only being `list`?

Same for `rtype` 

@saad-waqar